pytorch_lightning/plugins/sharded_plugin.py

```diff
@@ -84,3 +85,11 @@ def get_model_from_plugin(
         if isinstance(model, LightningShardedDataParallel):
             return model.module
         return model
+
+    def required_plugins(self, amp_backend: AMPType) -> list:
+        if amp_backend == AMPType.APEX:
+            raise MisconfigurationException(
+                'Sharded Plugin is not supported with Apex AMP, please using native AMP for 16-bit precision.'
+            )
+        if amp_backend == AMPType.NATIVE:
+            return [ShardedNativeAMPPlugin()]
```

Comment:
```suggestion
            return [ShardedNativeAMPPlugin()]
        return []
```
otherwise returns None